cocos/2d/components/ui-static-batch.ts

```diff
@@ -68,7 +69,7 @@ export class UIStaticBatch extends UIRenderer {
         this._color.set(value);
     }
 
-    get drawBatchList () {
+    get drawBatchList (): DrawBatch2D[] {
```

Comment:
Should add readonly?

Reply:
This fix results from automatic type inference, in this PR I prefer to keep the original type unchanged, otherwise this will change the original behavior